### PR TITLE
Practice 1 MPU6050

### PR DESCRIPTION
**Pull request by D-Alexandr**
Added first part practice 1 mpu6050. Script communicates to mpu6050 by i2c and display value temperature on Celsius and Fahrenheit

### CODE REVIEW

File: 04-mpu6050/scripts/mpu6050_temp.sh [hunk 1]
```diff
@@ -26,14 +24,17 @@ while [ true ]
 do
     temp_h=$(i2cget -y 0 $MPU6050_ADDR $TEMP_H)
     temp_l=$(i2cget -y 0 $MPU6050_ADDR $TEMP_L)
-    
+
     temp_adc_unsig=$((($temp_h << 8)+ $temp_l))
     temp_sig=$(((temp_adc_unsig & 0x7fff)- (temp_adc_unsig & 0x8000)))
     echo "Value ADC: $temp_sig"
 
     temp_c=$(echo " scale = 2; $temp_sig / 340 + 36.53" | bc)
```
Comment (yekovalyov) [inline 04-mpu6050/scripts/mpu6050_temp.sh]: Required resolution is 0.001. scale=2 provides only resolution 0.01

File: 04-mpu6050/scripts/mpu6050_temp.sh [hunk 2]
```diff
@@ -26,14 +24,17 @@ while [ true ]
...
     echo "Value temperature in Celsius: $temp_c"
 
+    temp_f=$(echo " scale = 2; (9/5 * $temp_c) + 32" | bc)
```
Comment (yekovalyov) [inline 04-mpu6050/scripts/mpu6050_temp.sh]: Required resolution is 0.001. scale=2 provides only resolution 0.01

**Review by yekovalyov — CHANGES_REQUESTED** [review]

**Comment by D-Alexandr** [comment]
mpu6050 sensitivity to temperature ±1/С°. Why do we need measurements with an accuracy of 0.001?

**Comment by yekovalyov** [comment]
> mpu6050 sensitivity to temperature ±1/С°. Why do we need measurements with an accuracy of 0.001?

Please read task and driver docs.

**Review by yekovalyov — CHANGES_REQUESTED** [review]
Issues in script implementation were addressed.
Only skeleton of module was implemented.

File: 04-mpu6050/module/mpu6050.c [hunk 3]
```diff
@@ -181,7 +182,7 @@ static ssize_t temp_show(struct class *class,
 {
 	mpu6050_read_data();
 
-	sprintf(buf, "%d\n", g_mpu6050_data.temperature);
+	sprintf(buf, "%d.%03d\n", g_mpu6050_data.temperature / shift_number, g_mpu6050_data.temperature % shift_number);
```
Comment (yekovalyov) [inline 04-mpu6050/module/mpu6050.c:185]: Please check your code style using script from Linux repository.
scripts/checkpatch.pl --no-tree -f *.c

**Review by yekovalyov — CHANGES_REQUESTED** [review]

**Review by yekovalyov — APPROVED** [review]

**Pull request merged**